contracts/allowListExamples/AllowListOffChainManaged.sol

```diff
@@ -65,7 +65,9 @@ contract AllowListOffChainManaged is Ownable {
                 r,
                 s
             );
-        if (owner() == signer) {
+        bytes memory allowListData =
+            EasyAuction(msg.sender).auctionAccessData(auctionId);
+        if (abi.decode(allowListData, (address)) == signer) {
```

Comment:
The code makes sense when the caller is `EasyAuction`, but not when the caller is something else.
I think it would be nice to read if a user is allowed from any contract, what about doing something like this:
1. renaming this function to `isAllowedBy`, which takes an extra `owner` input.
2. In these lines, you use `owner` instead of `msg.sender`.
3. `isAllowed` is defined as a new function that forwards everything to `isAllowedBy(msg.sender, ...)`

Reply:
Hm... I think its okay to not have it this generic and keep it simpler